app/scripts/HiGlassComponent.js

```diff
@@ -2806,7 +2806,7 @@ class HiGlassComponent extends React.Component {
       })
       .then(_json => ({
         id: _json.uid,
-        url: `${parsedUrl.origin}/app/?config=${_json.uid}`
+        url: `${parsedUrl.origin}/l/?d=${_json.uid}`
```

Comment:
Not sure I am a fan of this URL schema. Do we have to space or why did we decide not to use something more readable?
We can push this out but I would prefer `/link/?config=`